hubstorage/resourcetype.py

```diff
@@ -26,7 +28,9 @@ def _iter_lines(self, _path, **kwargs):
 
         r = self.client.request(**kwargs)
 
-        return r.iter_lines()
+        lines = r.iter_lines()
+        return (l.decode('utf8') if isinstance(l, six.binary_type)
+                else l for l in lines)
```

Comment:
@chekunkov, I couldn't reproduce the issue you had.

In `py3` branch:

```
In [20]: help()

Welcome to Python 2.7!  This is the online help utility.

(...)

In [21]: next(job.items.iter_json())

Out[21]: '{"_key":"1886/661/8/0","category":"\xe5\xae\xa0\xe7\x89\xa9\xe7\x94\xa8\xe5\x93\x81 > \xe7\x8b\x97\xe7\x94\xa8\xe5\x93\x81 > \xe7\x8b\x97\xe7\xb2\xae","_type":"DiscoveryItem","url_product_page":"http://www.amazon.cn/dp/B00CAO9M80","name":"american distribution and mfg co 00106 Nutri Life, 5 LB, Gold Plus Formula, Chicken, Duck, Fish, and Lamb Dog...","response_url":"http://www.amazon.cn/s/ref=sr_nr_n_13?fst=as%3Aoff&rh=n%3A118863071%2Cn%3A149410071%2Cn%3A149411071%2Ck%3Adog&keywords=dog&ie=UTF8&qid=1419962169&rnid=118864071","url":"http://www.amazon.cn/s?ie=UTF8&page=1&rh=n%3A149411071%2Ck%3Adog","country":"China","price_now":"135.21","image_urls":["http://ec4.images-amazon.com/images/I/51EKVXspZTL._AA160_.jpg"],"scrape_datetime":"2016-03-17T17:34:34Z","download_screenshot":false,"identifier_sku":"B00CAO9M80","page_number":1,"products_on_page":24,"country_code":"CN","page_item_number":1,"images":[],"shelf_type":"Regular","download_html":false}'
```

In `master` branch:

```
In [8]: help()

Welcome to Python 2.7!  This is the online help utility.

In [9]: next(job.items.iter_json())

Out[9]: '{"_key":"1886/661/8/0","category":"\xe5\xae\xa0\xe7\x89\xa9\xe7\x94\xa8\xe5\x93\x81 > \xe7\x8b\x97\xe7\x94\xa8\xe5\x93\x81 > \xe7\x8b\x97\xe7\xb2\xae","_type":"DiscoveryItem","url_product_page":"http://www.amazon.cn/dp/B00CAO9M80","name":"american distribution and mfg co 00106 Nutri Life, 5 LB, Gold Plus Formula, Chicken, Duck, Fish, and Lamb Dog...","response_url":"http://www.amazon.cn/s/ref=sr_nr_n_13?fst=as%3Aoff&rh=n%3A118863071%2Cn%3A149410071%2Cn%3A149411071%2Ck%3Adog&keywords=dog&ie=UTF8&qid=1419962169&rnid=118864071","url":"http://www.amazon.cn/s?ie=UTF8&page=1&rh=n%3A149411071%2Ck%3Adog","country":"China","price_now":"135.21","image_urls":["http://ec4.images-amazon.com/images/I/51EKVXspZTL._AA160_.jpg"],"scrape_datetime":"2016-03-17T17:34:34Z","download_screenshot":false,"identifier_sku":"B00CAO9M80","page_number":1,"products_on_page":24,"country_code":"CN","page_item_number":1,"images":[],"shelf_type":"Regular","download_html":false}'
```

I also created a test to make sure it always return a `unicode` string (`str` in py3):

![screenshot from 2016-04-21 17 55 58](https://cloud.githubusercontent.com/assets/1286651/14724165/d5d67360-07ea-11e6-89d9-aaafd536ac1c.png)